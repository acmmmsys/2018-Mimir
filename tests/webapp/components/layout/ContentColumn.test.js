import React from 'react';
import ContentColumn from '../../../../webapp/components/layout/ContentColumn'
import renderer from 'react-test-renderer';

let wrapper;

test('ContentColumn does render', () => {
    wrapper = shallow(
        <ContentColumn>

        </ContentColumn>
    );
    expect(wrapper).toMatchSnapshot();
});
