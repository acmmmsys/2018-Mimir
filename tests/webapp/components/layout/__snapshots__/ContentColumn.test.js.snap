// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ContentColumn does render 1`] = `
<GridColumn
  className="grid-column"
/>
`;
